src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/Common.ps1

```diff
@@ -270,6 +270,22 @@ function Get-SqlServerKeyVaultKeyTestEnvironmentParameters ()
 
 <#
 .SYNOPSIS
+Gets the virtual network subnet id used in the virtual network rules tests
+#>
+function Get-VirtualNetworkSubnetId ($vnetName)
+{
+	$vnetSubscriptionId = "d513e2e9-97db-40f6-8d1a-ab3b340cc81a"
```

Comment:
this will make the test unrecordable, so I will not approve until the test is recordable. you can either import vnet cmdlets or directly use the vnet sdk